My lottery project

bartick opened the pull request:
| Fixes       | # Issue No. |
|----|----|
| Lottery Project       |    #120    |    

# Description

Add a web3 based lottery project

## Type of change

Added a new project

- [x] New project (non-breaking change which adds functionality)

# How Has This Been Tested?

<img width="796" alt="Screenshot 2022-03-18 at 4 48 14 PM" src="https://user-images.githubusercontent.com/69100224/158993907-daf22eae-5ca5-49d8-b480-cdc9d4a4d99f.png">


# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have included a README.md file
- [x] My project follows this structure: Domain Folder/ Project Name Folder/ Project files and Readme.md files
- [x] I am a GSSoC'22 Participant

bartick commented:
Thanks for letting me work in this project. I have learned a lot of things

Please let me know If there is something wrong with this project

manisha-27 commented on React/lottery-react/README.md:
````diff
@@ -0,0 +1,24 @@
+# Lottery React
+
+## Description
+Connect with other people on a rinkeby network. Bet some eth tokens and enter the lottery.
+
+## Tech Stack
+web3, React
+
+## Show Steps on how to run in Local
+
+In `/web`
+Edit the .env.example to .env and provide all the data    
+Then
+```bash
+npm i
+node deploy.js
+npm test # should pass all test
+```
+
+In root
+```
+npm i
+npm start
+```
````
Visit #178 and add demo video.

bartick replied:
Yes I have added a working video Please look into it

heet-kumar reviewed (changes requested):
@bartick Hey Buddy Nice PR! and Project just make few changes like for creating the [Readme.md](http://readme.md/) file follow all the rules which are mentioned in [Rules.md](https://github.com/abhijeet007rocks8/Dev-Scripts/blob/main/Rules.md) .

Add this two sections (install dependencies and Quick Start) in [Readme.md](http://readme.md/) file in these particlular manner : 
<img width="1100" alt="Screenshot 2022-03-03 at 1 02 07 PM" src="https://user-images.githubusercontent.com/78725162/158399068-9c84a79b-17d7-47cc-b1ec-02ee536cfcd2.png">
<br />
**Quick Start** (how to run in local)
<hr />
<img width="1080" alt="Screenshot 2022-03-03 at 1 01 42 PM" src="https://user-images.githubusercontent.com/78725162/158399122-5fcc235c-b2af-4607-ac48-5e97b9481f4b.png">

heet-kumar commented:
@bartick add information like clone the reposittrory and and then how to run these project in local and that add these small changes and all . Then we are good to merge 👍 .

bartick commented:
@heet-kumar  Sorry for this late commit. But I have added the clone repo command. And the steps on how to run the code was already present in the Readme so I just updated it according to the Image hope it works now

heet-kumar reviewed (approved):
@bartick All looks good 👍 LGTM! 🥳
